server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandlerApi.java

```diff
@@ -0,0 +1,95 @@
+* Database Client which is then referenced throughout the file.
+*/
+
+public class StorageHandlerApi {
+
+  private static final Spanner spanner = StorageHandlerSetup.createSpannerService();
+  private static final DatabaseClient dbClient = StorageHandlerSetup.createDbClient(spanner);
+
+  /**
+  * Fetches a person by ID from the database and returns a {@link Person}.
+  *
+  * @param  userId    the user ID string used to query the Persons table from the database
+  * @return person    a {@link Person} object containing information from the database
+  */
+  public static Person fetchPersonFromId(String userId) {
+    Person person = StorageHandler.getPerson(dbClient, userId);
+    return person;
+  }
+
+  /**
+  * Fetches a club by ID from the database and returns a {@link Club}.
+  *
+  * @param  clubId    the club ID string used to query the Clubs table from the database.
+  * @return club      a {@link Club} containing information from the database
+  */
+  public static Club fetchClubFromId(String clubId) {
+    Club club = StorageHandler.getClub(dbClient, clubId);
+    return club;
+  }
+
+  /**
+  * Performs a transaction that adds a membership to the database.
+  * This method checks if a person is already a member of a club. If the person is
+  * already a member, it will throw an exception. Otherwise, it adds the row containing the
+  * user ID and club ID to the Memberships table.
+  *
+  * @param  userId      the user ID string used to perform the transaction
+  * @param  clubId      the club ID string used to perform the transaction
+  */
+  public static void performAddMembershipTransaction(String userId, String clubId) {
+    StorageHandler.getAddMembershipTransaction(dbClient, userId, clubId);
+  }
+
+  /**
+  * Performs a transaction that deletes a membership from the database.
+  * This method checks if a person is already a member of a club. If the person is
+  * not a member, it will throw an exception. Otherwise, it deletes the row
+  * containing the user ID and club ID from the Memberships table.
+  *
+  * @param  userId      the user ID string used to perform the transaction
+  * @param  clubId      the club ID string used to perform the transaction
+  */
+  public static void performDeleteMembershipTransaction(String userId, String clubId) {
+    StorageHandler.getDeleteMembershipTransaction(dbClient, userId, clubId);
+  }
+
+  /**
+  * Creates and returns a list of {@link Club}s depending on the user's membership status.
+  *
+  * @param  userId            the user ID string used to query and get a list of clubs
+  * @param  membershipStatus  the enum specifying whether the user is a member or not
+  * @return clubs             the list of {@link Club}s
+  */
+  public static List<Club> listClubsFromUserId(String userId, MembershipConstants.MembershipStatus membershipStatus) {
+    List<Club> clubs = StorageHandler.getListOfClubs(dbClient, userId, membershipStatus);
+    return clubs;
+  }
+}
```

Comment:
```suggestion
}

```
Newline at end of file.